src/universal/components/Dashboard/dashTimestamp.js

```diff
@@ -0,0 +1,33 @@
+const days = [
```

Comment:
In that case – let's just leave it for now and we can make it a util later. Easily enough refactored!